CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+This page contains instructions on how to propose and implement feature changes to Presto.
+
+# The Request for Comments
+
+## Overview of the process
+
+```mermaid
+graph TD;
+    id1["Create an RFC"]-->id2["Get feedback on the RFC"];
+    id2["Get feedback on the RFC"]-->id3["RFC is closed"];
+    id2["Get feedback on the RFC"]-->id4["RFC is merged"];
+    id4["RFC is merged"]-->id5["Create a Github issue linking the RFC"];
+    id5["Create a Github issue linking the RFC"]-->id6["Create PRs implementing the RFC (link to the issue)"];
+```
+
+## Step 1: Create an RFC
+RFCs are located in their own repository.
+
+To create one:
+
+1. Fork the https://github.com/prestodb/rfcs repository
+2. Copy the template file `RFC-0000-template.md` to `RFC-00xx-your-feature.md` and fill it out with your proposal.
+   The template is a guideline, feel free to add sections as appropriate
+3. You may also have the template simply link to another editor, like a Google Docs file, but please ensure that the
+   document is publicly visible.  This can make the template easier to add edit, but commenting doesn’t scale very well, so
+   please use this option with caution.
+
+## Step 2: Get Feedback on the RFC
+1. Submit a pull request [to the main repository](https://github.com/prestodb/rfcs) from your fork titled 
+`RFC-00xx-your-feature.md`
+2. Before your PR is ready for review,
+   [mark the PR as draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft).
+3. Once it’s ready for review, remove the draft label.
+4. In the description, include a short summary of your feature and a link to your RFC PR
+5. A committer will review your proposal.
+6. Build consensus. Those committers will review your PR and offer feedback. Revise your proposal as needed until
+   everyone agrees on a path forward. Additional forums you can share the proposal on include the
+   [the mailing list](https://lists.prestodb.io/g/presto-dev), and the
+   [Slack channel in #dev](https://communityinviter.com/apps/prestodb/prestodb). Tagging interested stakeholders
+   (identifiable via [CODEOWNERS](https://github.com/prestodb/presto/blob/master/CODEOWNERS)) can help with consensus building.
+
+_(Note: A proposal may get rejected if it comes with unresolvable drawbacks or if it’s against the long term plans of the Presto committers)_
+
+## Step 3: Implement your Feature
+1. If your RFC PR is accepted, the PR will be merged and markdown that adds the RFC will be merged it into the
```

Comment:
I think this statement can be simplified to "If your RFC is accepted, the PR will be merged."

Reply:
Sure